<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QDialog" name="StartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QPushButton" name="utworzProjekt_Button">
     <property name="text">
      <string>Utwórz projekt</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="wczytaj_pushButton">
     <property name="text">
      <string>Wczytaj</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="rokLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Rok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="miesiacLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Nazwa miesiąca</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="IloscDni_label">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ilość dni w miesiącu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pierwszyDzien__label">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Pierwszy dzień miesiąca, to</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="rokLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="miesiacLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IloscDni_lineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="pierwszyDzien_comboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>Poniedziałek</string>
         </property>
         <item>
          <property name="text">
           <string>Poniedziałek</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wtorek</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Środa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Czwartek</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Piątek</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sobota</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Niedziela</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="rozpocznij_Button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Rozpocznij</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
